spec/helpers/application_helper_spec.rb

```diff
@@ -1641,4 +1641,23 @@
       ).to eq('host_services')
     end
   end
+
+  describe "#multiple_relationship_link" do
+    context "When record is a Container Provider" do
```

Comment:
@abonas 
> I am wondering btw how is the test for "show" in containers provider is still working

we do not check the route, only the action.

if we wanted to check the route (but we do not), we will have to do:
`expect(get: '/ems_container/1').to route_to(controller: 'ems_container', action: 'show', id: '1')`